tedana/metrics/collect.py

```diff
@@ -16,13 +16,15 @@
 
 
 def generate_metrics(
+    *,
```

Comment:
I'm not sure why you added `*,` here. I don't think you're using it anywhere.

Reply:
The `*` forces function calls to use the input names (e.g., `func(a=5)` instead of `func(5)`)